lib/screenshot.dart

```diff
@@ -119,9 +119,9 @@ class ScreenshotController {
     }
 
     final RenderRepaintBoundary repaintBoundary = RenderRepaintBoundary();
-
-    Size logicalSize = ui.window.physicalSize / ui.window.devicePixelRatio;
-    Size imageSize = ui.window.physicalSize;
+    final Size rootSize = customSize ?? ui.window.physicalSize;
+    Size logicalSize = rootSize / ui.window.devicePixelRatio;
```

Comment:
for the use case of generating a consistent image of some document across phones, the logical size should be set to the same size as the image to ensure consistency across phones - have confirmed this through testing.